### ISSUE TITLE
petabytes does not work

### ISSUE THREAD

**Issue by az0** [az0/bleachbit#112]
This code shows "a lot" instead of petabytes, but it looks like you intended to show petabytes.  

Otherwise, the new code looks good, and I added unit tests for IEC units.

````
from bleachbit.FileUtilities import bytes_to_human;from bleachbit.Options import options
options.set('units_iec', False)
bytes_to_human(1000**5)
options.set('units_iec', True)
bytes_to_human(1024**5)
````

@tstenner , would you please take a look?

**Comment by tstenner** [comment]
My bad, seems like an off-by-one error got me.
`for exponent in range(0,len(prefixes)-1):` should be `for exponent in range(0,len(prefixes)):`.
With this, it works as intended: 

    In [3]: bytes_to_human(1000**5)
    Out[3]: '1PB'

**Comment by az0** [comment]
Fixed in cf7098705e57e1a3ee11ee1abf2c5df71986935e
